docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+the answers.
+
+* What type of forecasts and type of observations will be used? Will they be
+  gridded or point-based?  The METplus tools that can be used will vary
+  depending on the answer.  Here’s a matrix to help:
+
+.. role:: raw-html(raw)
+   :format: html	  
+
+.. list-table:: METplus Tools Decision Matrix
+  :widths: auto
+  :header-rows: 1
+		
+  * - 
+    - Gridded Forecast
+    - Point Forecast
+  * - Gridded :raw-html:`<br />`
+      Observation/Analysis
+    - Gen-Ens-Prod :raw-html:`<br />`
+      PCP-Combine :raw-html:`<br />`
+      Ensemble-Stat :raw-html:`<br />`
+      Grid-Stat :raw-html:`<br />`
+      Wavelet-Stat :raw-html:`<br />`
+      MODE :raw-html:`<br />`
+      MTD :raw-html:`<br />`
+      Series-Analysis :raw-html:`<br />`
+      Grid-Diag :raw-html:`<br />`
+      TC-Gen :raw-html:`<br />`
+      TC-RMW :raw-html:`<br />`
+    - Point-Stat (run with the Analyses :raw-html:`<br />`
+      as the “forecast” and Point Forecast :raw-html:`<br />`
+      as the “observation”)
+  * - Point Observations
+    - Point-Stat :raw-html:`<br />`
+      Ensemble-Stat
+    - Stat-Analysis (run by passing in MPR records) :raw-html:`<br />`
+      TC-Pairs :raw-html:`<br />`
+      TC-Gen :raw-html:`<br />`
+      TC-Stat
+
+      
+* What attributes of the forecast should be evaluated?
+  
+  * This refers to not only defining the forecast fields to be evaluated
+    but also the forecast characteristics such as bias, reliability,
+    resolution, and prediction of events.  It also means understanding
+    the nature of the forecast and observations. 
+
+Examples of the nature of fields to be evaluated
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Continuous fields – the values change at the decimal level.
+
+* Categorical fields – the values change incrementally most
+  likely as integers or categories.  Continuous fields can also be
+  turned into categorical fields via applying thresholds.
+  
+* Probability fields – the values represent the probability or
+  likelihood of an event occurring, usually represented by thresholds.
+   
+* Ensemble fields – are made up of multiple predictions either from
+  the same modeling system or multiple systems.
+
+Definitions of statistics categories associated with each type of field:
+ 
+* Continuous statistics - measures how the values of the forecasts
+  differ from the values of the observations.
+   
+  * METplus line types: SL1L2, SAL1L2, VL1L2, VAL1L2, CNT, VCNT.
+     
+  * METplus tools:
+      
+* Categorical statistics - measures how well the forecast captures events.
+   
+  * METplus line types: FHO, CTC, CTS, MCTC, MCTS, ECLV, TC stats,
+    ExtraTC stats, TC-Gen stats.
+    
+* Probability statistics - measures attributes such as reliability,
+  resolution, sharpness, and uncertainty.
+
+  * METplus line types: PCT, PSTD, PJC, PRC.
+     
+* Ensemble statistics - measures attributes as the relationship between
```

Comment:
This probably should also have an entry for line types: take a look at https://met.readthedocs.io/en/develop/Users_Guide/ensemble-stat.html#id1 for a list (ECNT, RPS, etc.).

Reply:
@TaraJensen, 
@j-opatz suggests an entry for line types here as well. Could you please define what that should be/look like?